### PR TITLE
feat: demoap setup

### PR DESCRIPTION
**Pull request by TapaiBalazs**
Added material design to the app.

See: https://stackblitz.com/github/thisdot/open-source/tree/feat/7-demoapp

### CODE REVIEW

File: apps/showcase/src/app/app.component.html [hunk 1]
```diff
@@ -1,13 +1,52 @@
...
+  </ng-container>
+  <ng-template #viewGithub>
+    <a
+      mat-button
+      color="accent"
+      class="header-github-link"
+      title="View the repository"
+      href="https://github.com/thisdot/open-source"
+      target="_blank"
+      >View on GitHub</a
```
Comment (dustinsgoodman) [inline apps/showcase/src/app/app.component.html:19]: This is showing red instead of white

File: apps/showcase/src/app/app.component.html [hunk 2]
```diff
@@ -1,13 +1,52 @@
...
 
-<router-outlet></router-outlet>
-
-<ng-container *tdRouteTag="'show'; else noShowTag">
-  <p>This text is only visible, if there is a 'routeTag.SHOW' in the route data</p>
-</ng-container>
-<ng-template #noShowTag>
-  <p>There is no show tag in this route's config</p>
-</ng-template>
+<mat-toolbar class="footer">© This Dot Labs 2021</mat-toolbar>
```
Comment (dustinsgoodman) [inline apps/showcase/src/app/app.component.html:52]:
```suggestion
<mat-toolbar class="footer">©This Dot Labs 2021</mat-toolbar>
```

File: apps/showcase/src/styles.scss [hunk 3]
```diff
@@ -1 +1,24 @@
 /* You can add global styles to this file, and also import other style files */
+html,
+body {
+  width: 100vw;
+  height: 100vh;
+  margin: 0;
+  padding: 0;
+  //background-color: #0a1930;
+  //color: #ecf1f7;
```
Comment (dustinsgoodman) [inline apps/showcase/src/styles.scss:9]: Remove unused styles please

File: apps/showcase/src/app/app.component.html (hunk 1 above)
Comment (TapaiBalazs) [reply apps/showcase/src/app/app.component.html:19]: fixed

File: apps/showcase/src/styles.scss (hunk 3 above)
Comment (TapaiBalazs) [reply apps/showcase/src/styles.scss]: removed

**Pull request merged**